_posts/2015-03-11-CocoaPods-0.36.markdown

```diff
@@ -99,6 +99,17 @@ To make use of that, the view component must live in an own framework.
 If you want to learn more about Frameworks, take a look into the [Framework Programming Guide](https://developer.apple.com/library/mac/documentation/MacOSX/Conceptual/BPFrameworks/Frameworks.html#//apple_ref/doc/uid/10000183-SW1).
 Even though this was not written specifically for the new Cocoa Touch Frameworks, how Apple calls them in their Xcode target template, those are mostly the same to the classic OS X frameworks, so this document is still a helpful introduction.
 
+## Private Pods and `source`
+
+On a last note, another change in this version is regarding private spec repositories.
+
+For those of you who use those to store your company's [private pods](http://guides.cocoapods.org/making/private-cocoapods.html), you previously had to `pod repo add` your private repository of podspecs and you were ready to go.  
+With 0.36.0 and forward, your `Podfile` needs to [explicity list all the spec repo sources](http://guides.cocoapods.org/making/private-cocoapods.html#thats-it) to use for finding your Podspecs (including CocoaPods's master spec repo and your own). This change is important for ensuring everybody in your team uses the same reference of specs whatever their local configuration is.
```

Comment:
@kyle ok with your phrasing.
But I feel like the "This is important because…" part is… well… important :smile: we should explain the rationale behind our choice.